MultiTarget/PackageReferences/WinAppSdk.props

```diff
@@ -1,6 +1,9 @@
 <Project>
   <ItemGroup>
-    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.5.240311000" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.6.240531000-experimental1" />
     <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
   </ItemGroup>
+  <ItemGroup Condition="'$(Configuration)' == 'Release'">
```

Comment:
This should not be conditional. Is this used by any libraries in the WCT too, or just the sample app?

Reply:
This is used by both Libraries and App Heads. If it's not meant to be conditional, we can fix that.

Reply:
Fixed in https://github.com/CommunityToolkit/Tooling-Windows-Submodule/pull/195/files/9349c6c5f77bdc3958f78e4533efe84f11258947#r1659294931